src/index.rst

```diff
@@ -616,6 +647,59 @@ Now ready to submit the workflow: ::
 
    $> bps submit test_sdf.yaml
 
+Submit a development workflow to USDF
+-------------------------------------
+
+To submit a development workflow to S3DF, please at first check `Submit a workflow to USDF`_.
+Here we only list the differences.
+
+Copy the environment setup script from cvmfs to your local directory and update the lsst
+setup part to your private repo: ::
+
+   $> latest=$(ls -td /cvmfs/sw.lsst.eu/linux-x86_64/panda_env/v* | head -1)
+   $> cp $latest/setup_panda_s3df.sh .
+   $> <update setup_panda_s3df.sh>
+   $> source ./setup_panda_s3df.sh
+
+``Note``: Make sure PanDA can read your private repo: ::
+
+   $> chmod g+rxs <your private development repo>
+
+For the submission yaml file ``test_usdf.yaml``, you need to change the ``runnercommand`` to point to your private development repo: ::
+
+   $> cat test_usdf.yaml
+   # An example bps submission yaml
+   # Need to setup USDF before submitting the yaml
+   # source setupUSDF.sh
+
+   LSST_VERSION: w_2022_32
+
+   includeConfigs:
+   - ${CTRL_BPS_PANDA_DIR}/config/bps_usdf.yaml
+
+   pipelineYaml: "${DRP_PIPE_DIR}/pipelines/HSC/DRP-RC2.yaml#isr"
+
+   payload:
+     payloadName: testUSDF
+     inCollection: "HSC/RC2/defaults"
+     dataQuery: "exposure = 34342 AND detector = 10"
+
+   # To override the 'loadLSST.bash' and 'setup lsst_distrib' with your development repo.
```

Comment:
This comment is a little confusing, am I supposed to change loadLSST.bash with my modified copy of the setup_panda_s3df.sh?   Or is it to "Add my software to the lsst stack, add the appropriate eups setup lines to the runnerCommand?